app/views/submissions/_form.html.erb

```diff
@@ -7,23 +7,23 @@
 
   <div class="form-field">
     <%= f.label :full_name %>
-    <%= f.text_field :full_name, maxlength: 50 %>
+    <%= f.text_field :full_name, maxLength: 50, required: true %>
   </div>
   <div class="form-field">
     <%= f.label :email %>
-    <%= f.text_field :email, maxlength: 50 %>
+    <%= f.email_field :email, maxLength: 50, required: true %>
   </div>
   <div class="form-field">
     <%= f.label :age %>
-    <%= f.number_field :age %>
+    <%= f.number_field :age, required: true, min: 0, max: 110 %>
   </div>
   <div class="form-field">
     <%= f.label :codecademy_username %>
-    <%= f.text_field :codecademy_username, maxlength: 50 %>
+    <%= f.text_field :codecademy_username, maxLength: 50, required: true %>
```

Comment:
Is having a codacademy account a requirement for attending rails girls?
In general +1 for PR